### PR TITLE
small typo change

### PR DESCRIPTION
**Pull request by Jeeppler**
bob -> Bob
Bobs -> Bob's

and fixed some sentence.

### CODE REVIEW

File: docs/updates/Crypto.md [hunk 1]
```diff
@@ -9,27 +9,27 @@ The generated public key is set as the client_id of the peer.
...
 For more info on the nospam see: Spam_Prevention.txt
         
 Bob receives the request and decrypts the message using the function crypto_box_open()
         
 If the message decrypts successfully:   
-If Alice is already in Bobs friends list: case 2  
+If Alice is already in Bob's friends list: case 2  
 If Alice is not in Bob's friends list and the nospam is good: Bob is prompt to add Alice and is shown the message from her.  
-If Bobs accepts Alice's friends request he adds her public key to his friends list.  
+If Bob accepts Alice friend request he adds her public key to his friends list.
```
Comment (JFreegman) [inline docs/updates/Crypto.md]: This should say:
>If Bob accepts Alice's friend request ...

**Comment by JFreegman** [comment]
much better!

**Comment by Jeeppler** [comment]
I am not a native english speaker, but it should be friend list and not friends list or?

**Comment by JFreegman** [comment]
friend list is correct.

**Pull request merged**